ui/src/components/Timeline2/TimelineList/TimelineListItem.scss

```diff
@@ -0,0 +1,83 @@
+@import 'app/variables';
+
+.TimelineListItem {
+  --background-opacity: 0.1;
+
+  position: relative;
+  overflow: hidden;
+  cursor: pointer;
+}
+
+.TimelineListItem > td:first-child {
+  border-left: 4px solid var(--timeline-item-color);
+  border-top-left-radius: $pt-border-radius;
+  border-bottom-left-radius: $pt-border-radius;
+}
+
+.TimelineListItem > td:first-child::before {
+  content: '';
+
+  position: absolute;
+  top: 0;
+  left: 0;
+
+  display: block;
+  width: 100%;
+  height: 100%;
+
+  background-color: var(--timeline-item-color);
+  opacity: var(--background-opacity);
+  border-radius: $pt-border-radius;
+}
+
+.TimelineListItem > td {
+  padding: var(--timeline-list-padding);
+}
+
+.TimelineListItem__date {
+  width: 1%;
+  white-space: nowrap;
+}
+
+.TimelineListItem__property {
+  width: 8rem;
+  color: $aleph-greyed-text;
+}
+
+// Double class is required in order to increase specificity over the
+// default Blueprint table styles.
+.TimelineListItem__actions.TimelineListItem__actions {
```

Comment:
This workaround is required to increase the specificity as otherwise the styles would be overridden by default Blueprint styles. Chaining the same class name twice keeps the selector as low as possible while increasing it as much as necessary.

[This section](https://cssguidelin.es/#keep-it-low-at-all-times) in the CSS Guidelines describes the reasoning behind the workaround in more detail. That web page is quite old know, and some of the topics discussed are solved by recent additions to the CSS standard, but it’s still a valuable resource imo.

For context, this workaround is necessary because custom component-specific styles end up *before* Blueprint’s default styles in our CSS bundle. This is something that I’d like to solve properly at some point (probably when also looking at the rest of the build toolchain), but I haven’t found the time to do so yet.